src/particles/ImpactXParticleContainer.cpp

```diff
@@ -137,7 +137,7 @@ namespace impactx
         pinned_tile.push_back_real(RealSoA::ux, px);
         pinned_tile.push_back_real(RealSoA::uy, py);
         pinned_tile.push_back_real(RealSoA::pt, pz);
```

Comment:
Both pt and pz appear here.
```suggestion
        pinned_tile.push_back_real(RealSoA::pt, pt);
```